src/validation.cpp

```diff
@@ -3881,6 +3942,17 @@ bool LoadBlockIndex(const CChainParams& chainparams)
     return true;
 }
 
+
+bool CChainState::ReceiveGenesisBlock(const CBlock &block, const CDiskBlockPos& blockPos, const Consensus::Params& consensusParams) {
+    CBlockIndex *pindex = AddToBlockIndex(block);
+    CValidationState state;
+    if (!ReceivedBlockTransactions(block, state, pindex, blockPos, consensusParams)) {
+        return error("LoadBlockIndex(): genesis block not accepted");
```

Comment:
In commit "Move a bunch of chainstate-manipulation functions into CChainState":

LoadBlockIndex should be ReceiveGenesisBlock or \_\_func__ now.